package.json

```diff
@@ -26,6 +26,8 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
+    "predeploy": "react-scripts build && cp build/index.html build/404.html",
```

Comment:
Would it alternatively be possible to let `react-router` handle the 404 behaviour when a user hits a route that is not recognised? This would then allow a more customisable 404 page within a react component that contains a user friendly message (e.g. "Sorry, this page doesn't exist").

Reply:
so that's basically what it's doing, because we're deploying on github it would use it's own 404 page instead of letting our app handle it (here we're copying our index.html and having that as the 404 page)

Reply:
👌 Yeah I had a feeling upon testing, that if there was a route which didn't match, it would use the github 404 page, but also it's been a while so may be misremembering